.changeset/funny-mirrors-explode.md

```diff
@@ -0,0 +1,8 @@
+---
+'@finos/legend-extension-dsl-persistence': major
+'@finos/legend-graph-extension-collection': minor
+'@finos/legend-manual-tests': minor
+'@finos/legend-studio-app': minor
+---
+
+Support persistence DSL in text mode in Studio
```

Comment:
Let's split this changeset.
1. `'@finos/legend-graph-extension-collection': patch`: with the message `Add DSL Persistence`
2. `'@finos/legend-manual-tests': patch` and 
`'@finos/legend-studio-app': patch` and `'@finos/legend-extension-dsl-persistence': patch`: with no message. I leave DSL Persistence with a `patch` because I think this is just the initial support.

Reply:
```suggestion
Add Persistence DSL
```